packages/grafana-ui/src/components/ConfirmButton/ConfirmButton.tsx

```diff
@@ -11,6 +11,13 @@ import { Button, ButtonVariant } from '../Button';
 export interface Props extends Themeable2 {
   /** Confirm action callback */
   onConfirm(): void;
+  /** Children */
+  children:
+    | React.ReactNode
```

Comment:
With this, it is technically still backwards compatible, right?

Reply:
not really tbh. before you could pass a custom button and that would be used instead of the component's internal button. now, whatever you pass as a child is wrapped in the internal button unless you pass a function. so visually it will almost certainly look wrong.

Reply:
Ah, makes sense. But is that the intended use, compared to just using a string or maybe just a button with an icon?